.golangci.yaml

```diff
@@ -7,12 +7,12 @@ issues:
     - path: pkg/metrics/types_windows.go
       linters:
         - revive
-        - gomnd
+        - mnd
```

Comment:
Necessary to address these errors:
```
level=warning msg="The linter 'gomnd' is deprecated (since v1.58.0) due to: The linter has been renamed. Replaced by mnd."
level=error msg="[linters_context] gomnd: This linter is fully inactivated: it will not produce any reports."
```